joomla/joomla-cms
# [4.0] Toolbar custom element improvements

### Summary of Changes

- Throw an error if the Joomla API hasn't been initiated
- Add `disconnectedCallback` method
- Use template literals
- eslint fixes

### Testing Instructions

1. Go to create a new article
2. Ensure the category selection dropdown still works as expected
